### PR TITLE
Company

### PR DESCRIPTION
**Pull request by emma-martin**
@minamrp please check if everything is ready to go!!
Component Company created, its ancestor is Experience. 
PropTypes included.

Thanks a lot :-)

### CODE REVIEW

File: src/components/Company/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Company extends Component {
+
+    render() { 
+        const {company} = this.props.company;
```
Comment (minamrp) [inline src/components/Company/index.js]: const{company} = this.props;

**Review by minamrp — CHANGES_REQUESTED** [review]
Review company/index.js

File: src/components/Company/index.js (hunk 1 above)
Comment (emma-martin) [reply src/components/Company/index.js]: @minamrp fixed the issue, see details:
From Experience component we receive "experience" as props:
```
{experience.map((item, index) => {
                        return (
                            <Company key={index} experience={item} />
                        );
                    })}
```

Passing it to Company as such:
`const {company} = this.props.experience;`

Thanks for your help!

**Review by minamrp — APPROVED** [review]
its ok!

**Pull request merged**